Makefile

```diff
@@ -138,6 +82,62 @@ $(error Can't build $(TARGET_ARCH) binaries on a $(HOST_ARCH) host)
 endif
 
 # ---------------------------------------------------------------------
+# Application configuration
```

Comment:
Huh, why this massive change to the `Makefile`? Or is it just GH going crazy again?